app/src/main/java/io/multy/util/Constants.java

```diff
@@ -90,4 +93,13 @@
     public static final String PUSH_TOPIC = "btcTransactionUpdate-";
     public static final String MULTY_IO_URL = "http://multy.io";
     public static final String CHAIN_ID = "CHAIN_ID";
+
+    public static final int DONATION_WITH_TRANSACTION = 10000;
```

Comment:
Should be DONATE_FEATURE not DONATION_FEATURE e.g. DONATE_PORTFOLIO or DONATE_CHARTS

Reply:
I will change it but i'm not sure that is good idea because this field has field  name, not field value